LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -1868,7 +1868,7 @@ SIDEARM_BEAM_ISOUNDRANGE = 15
 SIDEARM_BEAM_IENVIRONMENTDAMAGE = 1
 
 ;Weapon Range Tables
-VEKTOR_MAGNETIC_RANGE[0] = 0
+VEKTOR_MAGNETIC_RANGE[0] = 30
```

Comment:
You updated Vektor rifle, but not Bullpup or Autopistol.

Reply:
LOL Dangit. ok I'll fix those.

Reply:
This has been changed.